testgrid/cmd/configurator/main.go

```diff
@@ -85,6 +86,7 @@ func (o *options) gatherOptions(fs *flag.FlagSet, args []string) error {
 	fs.StringVar(&o.defaultYAML, "default", "", "path to default settings; required for proto outputs")
 	fs.BoolVar(&o.updateDescription, "update-description", false, "add prowjob info to description even if non-empty")
 	fs.StringVar(&o.prowJobURLPrefix, "prowjob-url-prefix", "", "for prowjob_config_url in descriptions: {prowjob-url-prefix}/{prowjob.sourcepath}")
+	fs.BoolVar(&o.strictUnmarshal, "strict-unmarshal", false, "whether or not we want to be strict when unmarshalling configs")
```

Comment:
nit: I would expect a more direct description of what "strictness" means. 

Ex. "If true, will not tolerate unknown YAML fields in TestGrid configuration"